tests/test_setup_utils.py

```
@@ -1608,6 +1635,90 @@ def test_install_service_update_successfully(
+    @patch("mmrelay.setup_utils.service_needs_update")
+    @patch("mmrelay.setup_utils.read_service_file")
+    @patch("mmrelay.setup_utils.get_user_service_path")
+    def test_install_service_update_prompt_eof_defaults_to_update(
+        self,
+        mock_get_path,
+        mock_read_service,
+        mock_needs_update,
+        mock_loginctl_available,
+        mock_service_enabled,
+        mock_service_active,
+        _mock_run,
```

Ruff (0.15.7)

[warning] Fixture `_mock_run` without value is injected as parameter, use `@pytest.mark.usefixtures` instead

(PT019)